### ISSUE TITLE
Flutter Colors should use const

### ISSUE THREAD

**Issue by danilofuchs** [amzn/style-dictionary#458]
![image](https://user-images.githubusercontent.com/17657014/92125588-93b2b180-edd5-11ea-9484-5d89c14fb0d3.png)

const constructors in Dart offer performance optimizations, and VSCode complains we are using `static const` without constant constructors

**Comment by dbanksdesign** [comment]
Forgive me I'm not super great with Flutter. I'm going off the [flutter example](https://github.com/amzn/style-dictionary/tree/main/examples/advanced/flutter) and the dart file it generated looks similar to the one you posted. Initially I was getting that same error, but I resolved it when I ran `pub get`. I think because initially the dart/flutter plugin in VSCode can't find the 'dart:ui' package so it doesn't know about the `Color` class. Let me know if that works!
